catalog/src/test/java/org/killbill/billing/catalog/TestCatalogService.java

```diff
@@ -33,7 +33,7 @@ public void testCatalogServiceDirectory() throws ServiceException, CatalogApiExc
         final DefaultCatalogService service = new DefaultCatalogService(new CatalogConfig() {
             @Override
             public String getCatalogURI() {
-                return "file:src/test/resources/versionedCatalog";
+                return "file:catalog/src/test/resources/versionedCatalog";
             }
```

Comment:
If I run it without "catalog/" it fails in my devbox :sweat:.
Can you please try it there?